### DISCUSSION TITLE
Replace SSL by HTTPS

### DISCUSSION THREAD

**Discussion by jf-viguier** [PrestaShop/PrestaShop#41064] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

SSL is deprecated since 2015:
https://en.wikipedia.org/wiki/Transport_Layer_Security

### Describe the solution you'd like

SSL should be replaced by TLS or a more generic term in UI and code.
Examples : PS_SSL_ENABLED configuration value, Tools::getShopDomainSsl, ShopUrl::domain_ssl

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by hibatallahAouadni** [comment]
Hello @jf-viguier 

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. 
So, we're waiting for your PR :muscle: 

Thanks!

**Comment by Hlavtox** [comment]
What about calling it "Enable HTTPS"?

And I would not be so sure about this change, everyone is calling it SSL certificate:
- https://www.hostgator.com/web-hosting
- https://www.siteground.com/web-hosting.htm
- https://www.godaddy.com/en-uk/hosting/web-hosting

**Comment by jf-viguier** [comment]
@Julievrz we could start by renaming "Activate SSL" by "activate HTTPS" in settings page ? WDYT ?

**Comment by jf-viguier** [comment]
@Hlavtox I've changed the title, Https sounds good to me.

**Comment by Julievrz** [comment]
> @Julievrz we could start by renaming "Activate SSL" by "activate HTTPS" in settings page ? WDYT ?

Let's use "Enable HTTPS" to try to be more consistent with the rest of the software. :-)
